01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+# !ruby
+# frozen_string_literal: true
+
+(1...20).each do |num|
+  if (num % 3) == 0 && (num % 5) == 0
```

Comment:
```suggestion
  if num % 3 == 0 && num % 5 == 0
```

こういうケースは`()`を使わないことが多いですね。
（デフォルトの演算子の優先順位では都合が悪いときに`()`を使う）